internal/podman/config.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2024 MongoDB Inc
```

Comment:
I suppose this is Podman?

Reply:
this license is ours, their files are not stamped with licenses but looking at https://github.com/containers/podman/blob/main/LICENSE it seems the same